product-service/src/functions/getProductsById/handler.ts

```diff
@@ -11,20 +11,30 @@ export const main: APIGatewayProxyEventHandler = async (event) => {
   try {
     const productId = event.pathParameters.productId;
 
+    console.log('Request: product by id', event.pathParameters.productId);
```

Comment:
**Question**: Why not productId if you already have such variable?

Reply:
return productId